sherpa-onnx/jni/jni.cc

```diff
@@ -76,11 +76,24 @@ class SherpaOnnx {
 
   bool IsReady() const { return recognizer_.IsReady(stream_.get()); }
 
-  void Reset(bool recreate) {
-    if (recreate) {
+  // If keywords is an empty string, it just recreates the decoding stream
+  // If keywords is not empty, it will create a new decoding stream with
+  // the given keywords appended to the default keywords.
+  void Reset(bool recreate, const std::string &keywords = {}) {
+    if (keywords.empty()) {
+      if (recreate) {
       stream_ = recognizer_.CreateStream();
-    } else {
+      } else {
       recognizer_.Reset(stream_.get());
+      }
```

Comment:
Could you indent the statements?

You can run
```
cd /path/to/sherpa-onnx
./scripts/check_style_cpplint.sh 
./scripts/check_style_cpplint.sh 1
./scripts/check_style_cpplint.sh 2
```
to check your code style.

Reply:
> Could you indent the statements?
> 
> You can run
> 
> ```
> cd /path/to/sherpa-onnx
> ./scripts/check_style_cpplint.sh 
> ./scripts/check_style_cpplint.sh 1
> ./scripts/check_style_cpplint.sh 2
> ```
> 
> to check your code style.

Done!
All code style checks passed